spec/unit/puppet_forge/v3/module_spec.rb

```diff
@@ -3,6 +3,35 @@
 describe PuppetForge::V3::Module do
   subject { described_class.new('authorname-modulename') }
 
+  let(:faraday_stubs) { Faraday::Adapter::Test::Stubs.new }
+
+  let(:conn) do
+    Faraday.new do |builder|
+      builder.adapter :test, faraday_stubs
+    end
+  end
+
+  before do
+    subject.conn = conn
+  end
+
+  let(:all_releases) do
+      {"releases" => [
+          {"version" => "1.0.1"},
+          {"version" => "1.0.0"},
+          {"version" => "0.9.0"}
+        ]
+      }
+  end
+
+  let(:deleted_releases) do
```

Comment:
This is nitpicky, but can we change this helper method name to :with_deleted_releases or :releases_including_deletes? The name caused me a bit of figuring out what the test was actually doing.